### PR TITLE
More WPS request tests

### PR DESCRIPTION
**Pull request by SPalominos**
Add the GetCapabilities and the Execute request test.
Uses xml resources to unmarshall the WPS Objects instead of creating by hand the Objects.

### CODE REVIEW

File: bundles/wps-service/src/main/java/org/orbisgis/wpsservice/Job.java [hunk 1]
```diff
@@ -43,4 +46,12 @@ public void setProcessState(ProcessState processState) {
     public ProcessState getState(){
```
Comment (ebocher) [inline bundles/wps-service/src/main/java/org/orbisgis/wpsservice/Job.java]: javadoc or  @Override ?

File: bundles/wps-service/src/main/java/org/orbisgis/wpsservice/Job.java [hunk 2]
```diff
@@ -43,4 +46,12 @@ public void setProcessState(ProcessState processState) {
...
         return state;
     }
+
+    public Map<URI, Object> getDataMap(){
```
Comment (ebocher) [inline bundles/wps-service/src/main/java/org/orbisgis/wpsservice/Job.java]: idem

File: bundles/wps-service/src/main/java/org/orbisgis/wpsservice/Job.java (hunk 1 above)
Comment (SPalominos) [inline bundles/wps-service/src/main/java/org/orbisgis/wpsservice/Job.java]: My bad, the PR with the javadoc is coming.

Comment (ebocher) [inline bundles/wps-service/src/main/java/org/orbisgis/wpsservice/Job.java]: No pb ;-)

**Comment by SPalominos** [comment]
I have committed the javadoc. If there is nothing else, it is ready for PR.

**Pull request merged**